common/travis/set_mirror.sh

```diff
@@ -11,5 +11,5 @@ for _i in etc/repos-remote.conf etc/defaults.conf etc/repos-remote-x86_64.conf ;
     sed -i "s:https:$TRAVIS_PROTO:g" $_i
 
     # Now set the mirror
-    sed -i "s:repo\.voidlinux\.eu:$TRAVIS_MIRROR:g" $_i
+    sed -i "s:alpha\.de\.repo\.voidlinux\.org:$TRAVIS_MIRROR:g" $_i
```

Comment:
shouldn't this be consistent with `$TRAVIS_MIRROR`?

Reply:
Uhhh no.  This entire block sets the repos to whatever $TRAVIS_MIRROR contains.